src/table/header-cell/th-element.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import clsx from 'clsx';
+import React from 'react';
+import { TableProps } from '../interfaces';
+import { getAriaSort, getSortingStatus } from './utils';
+import styles from '../body-cell/styles.css.js';
```

Comment:
`body-cell/styles.scss` file containing `.header-cell-*` classes is even more confusing than the previous revision.

If there are difficulties wrapping this refactoring, let's talk in DM